dlrs/main/classes/RollupCalculateJob.cls

```diff
@@ -113,19 +113,31 @@ public with sharing class RollupCalculateJob implements Database.Batchable<sObje
       // Retrieve organization info to put in the email
       Organization org = [SELECT Id, Name FROM Organization];
 
+      //Getting rollup name
+      String rollupName = rollup.isEmpty() ? '' : rollup[0].Name;
+
       // Errors are emailed to the user that scheduled this job
       Messaging.SingleEmailMessage mail = new Messaging.SingleEmailMessage();
       mail.setToAddresses(new List<String>{ UserInfo.getUserId() });
       mail.setSubject(
         String.format(
-          'Apex job {0} failed to update rollups in {1} ({2})',
-          new List<String>{ BC.getJobId(), org.Name, org.Id }
+          'Apex job "{0}" failed to update rollups in "{1} (ID: {2})"',
+          new List<String>{ BC.getJobId(), org.Name, org.Id, rollupName, lookupId }
```

Comment:
You've added additional available strings but they aren't formatted into the subject line. Could you look at how you'd like to get those added?

Maybe dropping the Apex Job, leave it in the body, but put the name in the subject? (I don't think we need the ID unless the rollup couldn't be found, something like a fallback)